.gitignore

```diff
@@ -19,3 +19,7 @@ coverage
 
 # bundled files
 lib/bundle.js
+
+package-lock.json
+
+front-end-testing/test/*
```

Comment:
Ah, this was a hangover from when I was taking screenshots for the readme. 
I ignored the contents of this directory so that I could gradually add tests in, run `npm run front-end-coverage` and screenshot and still commit as I went without adding the test.spec.js file I wanted the person doing the tutorial to make themselves.